src/Microsoft.ServiceFabric.Data.Indexing.Persistent/ReliableIndexedDictionary.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.ServiceFabric.Data.Indexing.Persistent
 {
-	internal class ReliableIndexedDictionary<TKey, TValue> : IReliableIndexedDictionary<TKey, TValue>
+	public class ReliableIndexedDictionary<TKey, TValue> : IReliableIndexedDictionary<TKey, TValue>
```

Comment:
Do you need the concrete class outside of this?  Any code outside this should be using the interface IReliableIndexedDictionary - casting to that, calling those methods, etc.